.github/workflows/release.yaml

```
@@ -45,32 +38,31 @@ jobs:
-        with:
-          go-version: "${{ env.GOVERSION }}"
-      - name: set environment
-        uses: HatsuneMiku3939/direnv-action@v1
+          sudo rm -rf ./* || true
+          sudo rm -rf ./.??* || true
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Setup environment
+        uses: ./.github/actions/setup-ubuntu
       - name: detect release tag
         run: echo "RELEASE_TAG=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:48: Double quote to prevent globbing and word splitting

(shellcheck)